### DISCUSSION TITLE
wifi reset

### DISCUSSION THREAD

**Discussion by Calamitou** [3kh0/ext-remover#407] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school somehow made it so that if you connect to a server, it resets the wifi every two minutes so I get logged out of games. Anyone know a fix???

**Comment by rwg100** [comment, marked as answer]
personal hotspot or connect to a different wifi network

**Reply by Calamitou** [reply]
i dont have any other networks :( and i dont have a personal hotspot

**Reply by Aka-but-nice** [reply]
U got a phone

**Reply by Calamitou** [reply]
ya

**Reply by Aka-but-nice** [reply]
alr, download  secure dns, then use it on the Chromebook

**Comment by rwg100** [comment]
use a proxy or vpn

**Reply by Calamitou** [reply]
... i would but too laggy

**Reply by rwg100** [reply]
its better than getting booted off any multiplayer game every 2 minutes

**Comment by Alex2Mayhem** [comment]
**_Try_** _**unplugging**_ **_the_** _**modem**_ **_for 30 seconds_** **_then plug it back in._**

**_If this issue persists please don't hesitate to call._**

_**Phone number**_ : **SCH-OOL-WIFI.**

**(its a fake number chill out github)** 



```
-Tech Support 2020-2023
```
